conbench/app/_plots.py

```diff
@@ -806,8 +857,9 @@ def time_series_plot(
     )
 
     p.legend.title_text_color = "darkgray"
-    p.legend.title = f"benchmark results: {len(samples)}"
+    p.legend.title = f"number of results: {len(samples)}"
     p.legend.location = "top_left"
+    p.legend.label_text_font_size = "12px"
```

Comment:
13px is default, i.e. this makes the legend contents a tiny little smaller. I felt motivated to do so because now we may have even one more entry in the legend.